wait/host_port_test.go

```diff
@@ -1,18 +1,25 @@
-package wait
+package wait_test
 
 import (
 	"context"
+	"errors"
 	"io"
 	"net"
 	"strconv"
 	"testing"
 	"time"
 
 	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/errdefs"
 	"github.com/docker/go-connections/nat"
 	"github.com/stretchr/testify/require"
 
 	"github.com/testcontainers/testcontainers-go/exec"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+var (
```

Comment:
This looks unrelated

Reply:
After moving `host_port_test.go` to `wait_test`, we need to import `wait` package to get `Internal()`, `ErrPortNotFound`, etc